### PR TITLE
Add more info to connect Desktop via TOR

### PR DESCRIPTION
**Pull request by ramontayag**

### CODE REVIEW

**Comment by ben-kaufman** [comment]
Hi, I see the PR is still marked as draft. Is it still the case?

**Comment by ramontayag** [comment]
Thanks for the reminder. Was waiting for it to be green then didn't check back. Marked it as ready anyway.

**Comment by ben-kaufman** [comment]
Thanks, there's an issue with Travis blocking new contributors sometimes, but since this is just a documentation change that shouldn't matter here.

File: docs/tor.md [hunk 1]
```diff
@@ -21,6 +21,18 @@ Alternatively, you can setup the Tor service. Install Tor on the same server tha
 * [Debian / Ubuntu](https://2019.www.torproject.org/docs/debian.html.en)
 * [macOS](https://2019.www.torproject.org/docs/tor-doc-osx.html.en)
 
+### Configure Specter Desktop
```
Comment (ben-kaufman) [inline docs/tor.md:24]:
```suggestion
### Configure Specter Desktop for Remote Instance
```

File: docs/tor.md [hunk 2]
```diff
@@ -21,6 +21,18 @@ Alternatively, you can setup the Tor service. Install Tor on the same server tha
...
+If you're using Specter Desktop (to use hardware wallets for example), then setup the app to connect to your remote Specter server.
```
Comment (ben-kaufman) [inline docs/tor.md:25]:
```suggestion
If you're using Specter Desktop on a remote machine and would like to connect to it over Tor (to use hardware wallets for example), then follow these steps to set up the app to connect to your remote Specter server.
```

**Pull request merged**